src/MLJGLMInterface.jl

```diff
@@ -87,17 +95,19 @@ Augment the matrix `X` with a column of ones if the intercept is to be
 fitted (`b=true`), return `X` otherwise.
 """
 function augment_X(X::Matrix, b::Bool)::Matrix
-    b && return hcat(X, ones(eltype(X), size(X, 1), 1))
+    b && return hcat(X, ones(float(Int), size(X, 1), 1))
```

Comment:
Why can we always use a `float` here?

Reply:
Because we can't always guarantee that `one(T)` would always be possible for every subtype of `AbstractFloat`. This is also what `StatsModels.jl` does.

Reply:
Great